test/validators.js

```diff
@@ -852,15 +852,26 @@ describe('Validators', function () {
             validator: 'isDate'
           , valid: [
                 '2011-08-04'
+              , '2011-09-30'
               , '04. 08. 2011.'
               , '08/04/2011'
               , '2011.08.04'
               , '4. 8. 2011. GMT'
+              , '2. 28. 2011. GMT'
+              , '2. 29. 2008. GMT'
+              , '2. 29. 1988. GMT'
               , '2011-08-04 12:00'
+              , '2/29/24'
```

Comment:
So I realized I've been doing date normalization the wrong way. There's built-in functions that use the UTC date. I've switch to using those. Tests should pass now.